.travis.yml

```diff
@@ -62,5 +62,5 @@ jobs:
       php: 7.1
       env: STATIC_ANALYSIS
       install: travis_retry composer install --prefer-dist
-      script: composer static-analysis
+      script: composer stan
```

Comment:
I'd say `phpstan` Technically phpcs is also static analysis.

Reply:
We have `lint` for that.

I have seen that shortcut used in a few places now, it seems to be common.

Reply:
Yes, not desired though :) https://twitter.com/OndrejMirtes/status/1189846443500154880

(As for lint, it's also wrong IMO.)

Reply:
I don't think naming the shortcuts just like the underlying tool is the greatest idea, as that couples them tightly.

Reply:
They can still come and go, be easily replaced etc. I don't see any coupling here 🤔.

I have not found generic name for phpstan nor phpcs so far. Both are linting and static analysis tools. So personally I came to conflusion to call them their names, rather than using inaccurate ones.

Reply:
That is a fine way of looking at it, too.

In the sake of saving time, how about we just go with the PR as is? :wink:

Reply:
@spawnia don't consider this a PR block 👋. Just wanted to express that the change in general goes in a good direction but in my opinion and by wording definition is not correct and might be misleading.

Reply:
BTW we often use `cs`/`codestyle` for phpcs on php-only projects where there are no other style checks.